src/main.js

```diff
@@ -82,10 +82,6 @@ export default function createApp({
 				if (eventAsString.indexOf("Cannot set property 'PX1065' of undefined") !== -1) {
 					return false;
 				}
-				// Skip sending failed Algolia analtyics related errors
-				if (eventAsString.indexOf('Before calling any methods on the analytics') !== -1) {
-					return false;
-				}
```

Comment:
As we wind down this implementation, we want to know if any of these are occurring so we can remove them.